.gitignore

```diff
@@ -42,4 +42,3 @@ lib/.crit-setup.files
 compel/include/asm
 include/common/asm
 include/common/config.h
-build/
```

Comment:
@adrianreber why `build/` has been removed in this commit?

Reply:
I am not sure any more but I think because new files in scripts/build were hidden from git. As I added files in scripts/build it seemed wrong.

Do you see problems with this change?

Reply:
The `build` folder is created by `make install`. This line was introduced with commit https://github.com/checkpoint-restore/criu/commit/967797a.

Reply:
I opened a PR to add `build/` in gitignore: https://github.com/checkpoint-restore/criu/pull/1609